.github/labeler.yml

```diff
@@ -18,6 +18,7 @@
   - docs/**
 'Changes Performance':
   - CI/physmon/reference/**
+  - Examples/Python/tests/**
```

Comment:
Wouldn't it be better to check if the hash text file changes specifically? A code change here does not necessarily mean any of the outputs change.

Reply:
right. Targeting now only the `root_file_hashes.txt` file